project/frontend/src/components/ResultsMap.js

```diff
@@ -59,23 +71,37 @@ function MapContainer(props) {
     );
   };
 
+  // TODO: add a spec here to explain what's happening.
   const createMarkers = () => {
-    if (!restaurants) {
+    if (restaurants.length === 0) {
       return;
     }
     let markers = [];
-    const numOfMarkers = Math.min(restaurants.length, 4);
-    for (let i = 0; i < numOfMarkers; i++) {
-      const coords = restaurants[i].key.latLngCoords;
-      const numInList = (i + 1).toString();
-      markers.push(
-        <MarkerIcon
-          lat={coords.lat}
-          lng={coords.lng}
-          id={i}
-          aria-label={'Your #' + numInList + ' Match'}
-        />
-      );
+    let coords = restaurants[currentCardIndex].key.latLngCoords;
+    markers.push(
+      <MarkerIcon
+        lat={coords.lat}
+        lng={coords.lng}
+        id={currentCardIndex}
+        aria-label={'Your #' + currentCardIndex + ' Match'}
```

Comment:
This will be "Your #0 Match" if currentCardIndex is 0. Maybe change to `aria-label={'Your #' + (currentCardIndex + 1) + ' Match'}`